src/config.json

```diff
@@ -19,7 +19,9 @@
         "extension_registry": "https://s3.amazonaws.com/extend.brackets/registry.json",
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
         "linting.enabled_by_default": true,
-        "build_timestamp": ""
+        "build_timestamp": "",
+        "healthDataServerURL" : "http://10.192.16.236:8080",
+        "healthDataTestServerURL" : "http://10.192.16.236:8080?test=true"
```

Comment:
I don't think this should be in the config file.  Unit tests should be able to pass in an override URL programmatically... see UpdateNotification-test or ExtensionManager-tet for examples of other places we've done that.